lib/beaker-hostgenerator/data.rb

```diff
@@ -1,11 +1,21 @@
 module BeakerHostGenerator
   module Data
 
-    # Pull various informations out of the environment.
-    PE_VERSION=ENV['pe_version']
-    PE_FAMILY=ENV['pe_family']
-    PE_UPGRADE_VERSION=ENV['pe_upgrade_version']
-    PE_UPGRADE_FAMILY=ENV['pe_upgrade_family']
+    def pe_version
+      ENV['pe_version']
```

Comment:
We expect these values to change in the context of both test fixture generation and test fixture validation:
https://github.com/puppetlabs/beaker-hostgenerator/blob/qeng-3370/spec/beaker-hostgenerator/generator_spec.rb#L106-L113
https://github.com/puppetlabs/beaker-hostgenerator/blob/qeng-3370/test/util/generator_helpers.rb#L34-L40

Otherwise I probably would probably hate Ruby slightly less right now.